### PR TITLE
Enable logging out

### PR DESCRIPTION
**Pull request by huwshimi**
Done:
- Enable logging the user out. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1526

QA:
- Log into the UI.
- Click Logout in the top right.
- You should see the login form.
- Log in again, you should be correctly logged in again.

### CODE REVIEW

File: ui/src/app/base/components/Header/Header.js [hunk 1]
```diff
@@ -213,7 +215,11 @@ export const Header = () => {
               <li className="p-navigation__link" role="menuitem">
                 <a
                   className="p-dropdown__item"
-                  href={generateURL("accounts/logout/")}
+                  href="##"
```
Comment (steverydz) [inline ui/src/app/base/components/Header/Header.js]: Why `##` rather than `#`?

Comment (huwshimi) [reply ui/src/app/base/components/Header/Header.js]: To satisfy the linter :)

Comment (steverydz) [reply ui/src/app/base/components/Header/Header.js]: Ah ok. Maybe we should override that rule if we have to hack around it?

Comment (huwshimi) [reply ui/src/app/base/components/Header/Header.js]: Yeah good idea, I've filed: https://github.com/canonical-web-and-design/maas-ui/issues/284

**Pull request merged**